src/Lib/Managers/HoverManager.vala

```diff
@@ -103,7 +86,7 @@ public class Akira.Lib.Managers.HoverManager : Object {
             canvas.window.event_bus.hover_over_item (item);
         }
 
-        set_cursor_for_nob (Managers.NobManager.Nob.NONE);
+        return true;
```

Comment:
It would be helpful to get the meaning of the return (true or false) in the comment of this method, just to know what they mean and what they affect.

Reply:
Made it void. In the future we can see if it needs to return whether it was absorbed.